common/artifacts/000_wc_artifacts.txt

```diff
@@ -818,6 +1161,107 @@ chalice_of_trollbanes = {
 
 	picture = "GFX_inv_misc_trophy_argent"
 }
+orb_of_corruption = {
+	quality = 4
```

Comment:
Why it's legendary? It stopped appearing in the lore after Warcraft 3. I would give it 3 quality.

Reply:
reduced to 3